.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,11 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.12.* *_cpython
+python_min:
+- '3.9'
```

Comment:
I think the answer is probably "yes" but just wanted to double check that this `python_min` is unrelated to the `python_min` over in `recipe/meta.yaml` (given the 3.9 vs. 3.10 mismatch)